Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM nvidia/cuda:11.6.0-devel-ubuntu20.04 AS tm_base_env
 # Copied out of anaconda's dockerfile
 ARG MINICONDA_VERSION=4.6.14
 ARG MAKE_VERSION=4.2.1-1.2
-ARG GIT_VERSION=1:2.25.1-1ubuntu3.3
+ARG GIT_VERSION=1:2.25.1-1ubuntu3.4
```

Comment:
What about doing something like `1:2.25.1-*` that way we get the same version of git, but don't have to worry about the specific apt versions disappearing (which they seem to do)?